### PR TITLE
Multiple background images

### PR DESCRIPTION
**Pull request by fatihbatal**
:tada: :sparkles:

Support for multiple background images

This commit would give lozad.js the ability to support multiple background images as
```html
<div style="background: url( 'path/to/main/image' ), url( 'path/to/fallback/image' );"></div>
```

The initialization will be not so different from existing way; simply add multiple paths/urls separated by commas:
```html
<div class="lozad" data-background-image:"path/to/main/image,path/to/fallback/image"></div>
```

Then, lozad.js will split the data attribute by comma and join the parts again to match the CSS syntax.

This change is backward compatible; that is, it does not break existing code (of users of lozad.js).

### CODE REVIEW

**Comment by ApoorvSaxena** [comment]
@fatihbatal Readme update and inclusion in Demo page will be super helpful as well

File: dist/lozad.es.js [hunk 1]
```diff
@@ -94,7 +94,7 @@ const getElements = (selector, root = document) => {
 };
 
 function lozad (selector = '.lozad', options = {}) {
-  const {root, rootMargin, threshold, load, loaded} = {...defaultConfig, ...options};
```
Comment (ApoorvSaxena) [inline dist/lozad.es.js]: any specific reason to remove this up?

Comment (fatihbatal) [reply dist/lozad.es.js]: I didn't remove that, not manually/deliberately at least.
After changing the relevant part (line 48 to 50 in lozad.js/src/lozad.js), 
I've issued following commands:
```bash
npm run lint
npm run test
npm run build
```
Then pushed the PR up simply by
```bash
git push --set-upstream origin multiple-background-images
```

Nothing else, I'm positive.
I'm not sure about what went wrong.

It's hard to deal with a newbie, huh?

**Comment by fatihbatal** [comment]
> @fatihbatal Readme update and inclusion in Demo page will be super helpful as well

Both updated and pushed as PR.

**Pull request merged**